ex2_hal/athena/equipment_handler/source/housekeeping_athena.c

```diff
@@ -33,33 +32,26 @@ const uint8_t software_version = 3;
  * @return
  * 		0 for success. other for failure
  */
-int HAL_get_temp_all(long *temparray) {
+int HAL_get_temp_all(long *MCU_core_temp_add, long *converter_temp_add) {
 #if ATHENA_IS_STUBBED == 1
     return 0;
 #else
     return gettemp_all(temparray);
 #endif
 }
-
 /**
  * @brief
  *      Getter to Athena uptime
  * @return
  * 		integer OBC_uptime to store Athena uptime
  *      Seconds = OBC_uptime*10
- *      Max = 655350 seconds (7.6 days)
  */
-uint16_t Athena_get_OBC_uptime() {
+uint32_t Athena_get_OBC_uptime() {
```

Comment:
```suggestion
uint32_t get_obc_uptime_seconds() {
```

Reply:
"get" at the front is slightly preferable

Reply:
maybe just preference tho

Reply:
Get at the front is better